heroes.js

```diff
@@ -84,4 +84,5 @@
 // 			array.splice(index, 1);
 // 		}
 // 	});
-// }
\ No newline at end of file
+// }
+//new comment for new branch :)
```

Comment:
The rest of the code is not in the pull request therfore i will write it here:
Next time please think about the maintenance side. devide the logger class to ConsoleLogger and FileLogger that inherite from base Logger class(that implements LoggerInterface).